components/contributors.tsx

```diff
@@ -0,0 +1,124 @@
+import { Link } from '@/components/link/link';
+
+const ROLE_PREFIX_REGEX = /^https?:\/\/credit\.niso\.org\/contributor-roles\//i;
```

Comment:
Once we build out that ContributorRole table I will update the GraphQL schema to fetch the label and URL so that we can get rid of some of this logic :)

Reply:
Sounds good. I'll work on creating the Contributor Role Management page after I merge this.